app/controllers/evidence_controller.rb

```diff
@@ -23,6 +23,12 @@ def result
     evidence_result
   end
 
+  def summary
+    evidence_view
+    evidence_overview
+    evidence_result
+  end
```

Comment:
This begs to be cleaned up.

I've got to 'jump' up and down to figure out why and what is going on. The intent is not clear.